types/camelcase-keys-deep/camelcase-keys-deep-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as camelcaseKeysDeep from "camelcase-keys-deep";
```

Comment:
please use `import camelcaseKeysDeep = require('camelcase-keys-deep')` to match the `export =` syntax.

Reply:
This is changed in the new commit, and i also changed some other small things.